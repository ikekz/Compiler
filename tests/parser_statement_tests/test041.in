function f (a: integer; b: double): integer;
begin
end;
begin
	f(5.0, 5);
end.