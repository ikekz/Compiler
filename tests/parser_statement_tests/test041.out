(5,4) Error: Incompatible types for arg 1: got "double" expected "integer"